--- uinput.cpp.bak	2014-11-04 22:37:17.890333000 +0000
+++ uinput.cpp	2014-11-04 22:37:47.750333000 +0000
@@ -65,9 +65,7 @@
     memset(&dev_mk, 0, sizeof(dev_mk));
 
     if (DEV_TYPE == DEV_TYPE_SIXAXIS) {
-        strcpy(dev_name, "PLAYSTATION(R)3 Controller (");
-        strcat(dev_name, mac);
-        strcat(dev_name, ")");
+        strcpy(dev_name, "Sony PLAYSTATION(R)3 Controller");
         snprintf(dev.name, sizeof(dev.name), "%s", dev_name);
         dev.id.vendor = 0x054c;
         dev.id.product = 0x0268;
