--- bluetooth.cpp.orig	2016-06-14 23:47:07.456168798 +0100
+++ bluetooth.cpp	2016-06-14 23:48:54.365436856 +0100
@@ -233,7 +233,7 @@
             char bda[18];
             ba2str(&addr_dst, bda);

-            const char* uinput_sixaxis_cmd = "/usr/sbin/sixad-sixaxis";
+            const char* uinput_sixaxis_cmd = "/usr/bin/sixad-sixaxis";
             const char* debug_mode = debug ? "1" : "0";

             const char* argv[] = { uinput_sixaxis_cmd, bda, debug_mode, NULL };
