--- sixad-sixaxis.cpp	2017-08-25 22:49:46.439051211 +0100
+++ sixad-sixaxis.cpp.fix	2017-08-25 22:58:33.684511568 +0100
@@ -254,6 +254,14 @@
         return 1;
     }
 
+    timeval tv;
+    tv.tv_sec = 0;
+    tv.tv_usec = 100000;  // 0.1 sec.
+ 
+    if (-1 == setsockopt(csk, SOL_SOCKET, SO_RCVTIMEO, &tv, sizeof(tv))){
+	syslog(LOG_ERR, "could not set socket read timeout, error: %s", strerror(errno)); 
+    }
+
     enable_sixaxis(csk);
     led_n = set_sixaxis_led(csk, settings.led, settings.rumble.enabled);
 
@@ -352,23 +360,13 @@
         uinput_close(ufd->mk, debug);
     }
 
+    delete ufd;
+
     do_rumble(csk, 10, 0xff, 0xff, 0x01);
-    usleep(10*1000);
 
     shutdown(isk, SHUT_RDWR);
     shutdown(csk, SHUT_RDWR);
 
-    delete ufd;
-
-    // hack for force disconnect
-    char cmd[32] = { 0 };
-    strcpy(cmd, "hcitool dc ");
-    strcat(cmd, mac);
-
-    usleep(10*1000);
-    syslog(LOG_INFO, "Force disconnect of \"%s\"", mac);
-    system(cmd);
-
     if (debug) syslog(LOG_INFO, "Done");
 
     return 0;
