--- uinput.cpp.orig	2016-06-14 23:20:24.677317392 +0100
+++ uinput.cpp	2016-06-14 23:20:55.017103335 +0100
@@ -65,18 +65,14 @@
     memset(&dev_mk, 0, sizeof(dev_mk));
 
     if (DEV_TYPE == DEV_TYPE_SIXAXIS) {
-        strcpy(dev_name, "PLAYSTATION(R)3 Controller (");
-        strcat(dev_name, mac);
-        strcat(dev_name, ")");
+        strcpy(dev_name, "PLAYSTATION(R)3 Controller");
         snprintf(dev.name, sizeof(dev.name), "%s", dev_name);
         dev.id.vendor = 0x054c;
         dev.id.product = 0x0268;
         dev.id.version = 0x0100;
         dev.id.bustype = BUS_VIRTUAL;
     } else if (DEV_TYPE == DEV_TYPE_REMOTE) {
-        strcpy(dev_name, "PLAYSTATION(R)3 Remote (");
-        strcat(dev_name, mac);
-        strcat(dev_name, ")");
+        strcpy(dev_name, "PLAYSTATION(R)3 Remote");
         snprintf(dev.name, sizeof(dev.name), "%s", dev_name);
         dev.id.vendor = 0x054c;
         dev.id.product = 0x0306;
@@ -90,9 +86,7 @@
         dev.id.version = 0x0100;
         dev.id.bustype = BUS_VIRTUAL;
     } else {
-        strcpy(dev_name, "Unkown Device (");
-        strcat(dev_name, mac);
-        strcat(dev_name, ")");
+        strcpy(dev_name, "Unkown Device");
         snprintf(dev.name, sizeof(dev.name), "%s", dev_name);
         dev.id.vendor = 0x000;
         dev.id.product = 0x000;
